admin/class-admin.php

```diff
@@ -14,6 +14,22 @@ class WPSEO_Admin {
 	private $options;
 
 	/**
+	 * Array with all pages of the plugin.
+	 *
+	 * @var array
+	 */
+	private $wpseo_pages = array(
```

Comment:
Consider making this a constant, allowing other classes to also access this list elsewhere.